### PR TITLE
gevent-websocket (#311)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Solve Issue #311

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -28,6 +28,7 @@ commands =
 platform = linux
 commands =
     pipenv install --dev --skip-lock
+    pipenv run pip uninstall gevent gevent-websocket -y
```
Comment (FredLL-Avaiga) [inline tox.ini:31]: I thought we could use pipenv uninstall gevent ... (without run pip) ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: tox.ini (hunk 1 above)
Comment (dinhlongviolin1) [reply tox.ini:31]: it is equivalent in this case since we won't be using piplock

File: taipy/gui/gui.py [hunk 2]
```diff
@@ -1270,6 +1271,11 @@ def run(
                 Configure TLS to serve over HTTPS. Can be an ssl.SSLContext object, a (cert_file, key_file) tuple to
                 create a typical context, or the string 'adhoc' to generate a temporary self-signed certificate.</br>
                 The default value is None.
+            async_mode (Optional[str]): A configuration of Flask-SocketIO. Valid async modes are `threading`, `eventlet`,
```
Comment (FabienLelaquais) [inline taipy/gui/gui.py]: I suspect we are going to need more explanations on this one.
Let's talk about it.

**Review by FabienLelaquais — APPROVED** [review]
We will need a bit more doc on this Long.
Good work though!

**Pull request merged**